### ISSUE TITLE
Keep gaps at the top the way they are and don't rearrange them to fit the box 

### ISSUE THREAD

**Issue by marczking** [metafizzy/packery#176]
Please bear with me on this one:

I am using packery for an infinite-scrolling page. Pretty quickly a lot of elements can get appended. When then resizing an element it becomes slower and slower:

1. Click resize
2. Wait 1-3 Seconds
3. Item gets resized

So a way to go is to remove items at the top as new ones get added. This helps keep performance consistent. And when a User scrolls up I also just lazyload the previous results and prepend these.

And here comes my issue I am having:

When after you remove elements at the top and then call layout (f.ex. for a resize) all elements shift to fit the box at the top.

**Remove and append some elements and call *`.packery('layout')`*.**
![bildschirmfoto 2014-05-11 um 15 51 03](https://cloud.githubusercontent.com/assets/2788504/2938559/658fc492-d913-11e3-9974-11cf74b4f038.png)
With current behaviour you get:
![bildschirmfoto 2014-05-11 um 15 51 14](https://cloud.githubusercontent.com/assets/2788504/2938567/aafe96d4-d913-11e3-855b-f432ebc938e0.png)
I actually want to achieve this behaviour (keep those gaps at the top!)
![bildschirmfoto 2014-05-11 um 16 07 26](https://cloud.githubusercontent.com/assets/2788504/2938592/b27e0366-d915-11e3-90a8-a50b3a37ed9b.png)

Basically I want the exact same behaviour at the top as I have now at the bottom when appending new items. They should not be forced to realign at the top / left (0 / 0) to right. Just like when appending new items the `container` also does not always perfectly fit all corners.

I can not think of a way to achieve this. Every time I call `packery('layout')` my elements get realigned at the top corner.

I hope this is comprehensible and if you have a suggestion how I can work this out. Thanks!

Here is my pen: http://codepen.io/marczking/pen/HDxij

**Comment by desandro** [comment]
It's been a while and this issue didn't get enough support. I'm closing this one. :door:

**Comment by marczking** [comment]
Too bad :( but thanks anyways!
Am 08.03.2015 23:00 schrieb "David DeSandro" <notifications@github.com>:
